metricbeat/module/elasticsearch/pending_tasks/_meta/fields.yml

```diff
@@ -0,0 +1,25 @@
+- name: cluster.pending_tasks
```

Comment:
I would prefer to call this `pending_task`instead of `pending_tasks`. Reason is that the document ending up in Elasticsearch is a single task and not multiple.